SageMaker/02_classification.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>Maybe add a brief comment what this is doing. E.g. "As in the previous chapter we setup a bucket and get the SageMaker roles and session."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>